torch/_inductor/graph.py

```diff
@@ -208,6 +208,8 @@ def __init__(
         self.name = "GraphLowering"
         self.cpp_wrapper = cpp_wrapper
         self.aot_mode = aot_mode
+        self.aot_constant_curr_offset: int = 0
+        self.aot_constant_offset: List[Tuple[str, int]] = []
```

Comment:
Instead of attaching them in the general ```GraphLowering``` class, I am wondering if there would be another way to hold ```aot_constant_curr_offset``` and ```aot_constant_offset```. These two fields look too special to be placed here. 

Any thoughts or suggestion? @desertfire @eellison @jansel Thanks!

Reply:
Perhaps the wrapper code object?

Reply:
Done. I used `OrderedDict` instead so we could just iterate through the constant.